### DISCUSSION TITLE
Setup without typical energy buy and sell prices

### DISCUSSION THREAD

**Discussion by Xitee1** [jmcollin78/solar_optimizer#23] [Q&A]
Why are the energy buy and sell prices required?
In my case I don't have a typical energy buy and sell situation.
Instead we have a contract that gives us a specific amount of free kWh per year from the grid that we can use without paying anything.
Only after this free limit has been exceeded, we have to pay. If we use less than the free amount we get paid back the unused kWh.
On the other side we get nothing out of the exported energy (only a few cents per month because of some regulations).

With what values should I set up the integration?

**Comment by jmcollin78** [comment]
They are required because their are part of the cost function which have to be minimized. If you don't have, just put an input_number with 1. It have an effect only if the difference between sell and buy cost is important or if you have a many Watt installed.

**Comment by jmcollin78** [comment]
Note: this should be more a discussion and not an issue.

**Comment by Xitee1** [comment]
Thank for the info. I'll set them to 1 then.

I know that this should be rather an discussion instead of an issue, but it seems like you haven't enabled discussions yet.

**Comment by jmcollin78** [comment]
> like you haven't enabled discussions yet.

yes. Indeed. I will fix that immediatly
